nav2_bt_navigator/behavior_trees/follow_point.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+  This Behavior Tree follows a dynamic pose to a certain distance
+-->
+
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <PipelineSequence name="NavigateWithReplanning">
+      <RateController hz="1.0">
+        <GoalUpdater input_goal="{goal}" output_goal="{updated_goal}">
+          <TruncatePath distance="1.0" input_path="{path}" output_path="{truncated_path}">
```

Comment:
Semantically, wouldn't it make more sense if TruncatePath was an action after ComputePathToPose? @naiveHobo what do you think?

It seems to me like both of these don't really need to be decorators, but I suppose it works. @fmrico can you explain maybe why you think of it this way? Maybe its a 'me' thing.

Reply:
I implemented it as a decorator because it modifies the output/input of an action node. It has no sense without the action nodes that it modifies. Am I wrong?

Reply:
It just seems so paired to this particular action node that it itself seems to be computing an action. I could go either way on it, there are many ways to implement the same thing.

Reply:
I'm also on the edge here. The node only makes sense when it's used with `ComputePathToPose`. I can see it as an action node that is ticked in sequence after `ComputePathToPose` is ticked successfully.

One argument to justify `TruncatePath` to be an action node instead of a decorator is that technically it is supposed to work on the output of `ComputePathToPose`. Having it as a decorator forces it to tick _before_ a new path is computed. In my opinion, the natural flow would be to get a goal, compute a new path to the goal, truncate the path, and execute.

Reply:
OK. So do you think we should change or leave it?

Reply:
I think making it an action node such that it ticks after `ComputePathToPose` returns success makes more sense. That way this node isn't ticked in every cycle, it's only ticked once a new path is available. More natural fit in my opinion.

Reply:
Ok. Changing to action node